rust-actix/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM alpine:edge as BUILDER
```

Comment:
I did not add a Dockerfile to the ExpressJs but just used the Docker compose.

Reply:
You need to build the image some way and to do that you need the docker file.